pa_config/rca.conf

```diff
@@ -58,19 +58,19 @@
   },
 
   "muted-rcas": [
-    "HotShardRca",
-    "HotShardClusterRca",
-    "HeapAllocRateByShardTemperatureCalculator",
-    "HeapAllocRateByShardAvgTemperatureCalculator",
-    "HeapAllocRateShardIndependentTemperatureCalculator",
-    "HeapAllocRateTotalTemperatureCalculator",
-    "CpuUtilByShardsMetricBasedTemperatureCalculator",
-    "AvgCpuUtilByShardsMetricBasedTemperatureCalculator",
-    "ShardIndependentTemperatureCalculatorCpuUtilMetric",
-    "TotalCpuUtilForTotalNodeMetric",
-    "CpuUtilDimensionTemperatureRca",
-    "HeapAllocRateTemperatureRca",
-    "NodeTemperatureRca",
-    "ClusterTemperatureRca"
-   ]
+  //   "HotShardRca",
```

Comment:
should we keep those RCAs as muted ?

Reply:
Good catch. It was for testing. Didn't mean for this to be checked in